ui/src/app/components/help-sidebar.tsx

```diff
@@ -111,14 +111,18 @@ const styles = reactStyles({
   },
   runtimeStatusIcon: {
     width: '.5rem',
-    height: '.5rem',
-    margin: '0px .1rem .1rem auto',
-    alignSelf: 'flex-end'
+    height: '.5rem'
   },
   runtimeStatusIconOutline: {
     border: `1px solid ${colors.white}`,
     borderRadius: '.25rem',
   },
+  runtimeStatusIconContainer: {
+    // This height properly aligns the status indicator vertically.
+    height: '.9rem',
```

Comment:
There's probably a cleaner way of doing this. I wasn't really able to understand how the  svg was positioning itself relative to the div, and it seemed like I was going down a bit of a rabbit hole. Let me know if you have another idea for achieving this.